tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,97 @@
+
+## Records update as data changes
+
+Blocks in Eve react automatically to changes in data. When a record changes, any bound records are automatically updated. Let's search for the current time, and display it in a div:
+
+```eve
+search
+  [#time seconds]
+
+bind 
+  [#ui/text text: seconds]
+```
+
+As the time changes, the output updates to reflect the current state of the `#time` record. Records can be committed instead of bound, but the behavior is a little different -- committed records persist until they are removed explicitly. Try changing `bind` to `commit` in the above block and see what happens.
+
+## Reacting to events
+
+Let's draw a button on the screen:
+
+```eve
+commit
+  [#ui/button #increment text: "+1"]
+```
+
+When you click anywhere on the screen, Eve creates an `#event/click` record representing the click. You can react to clicks on the `#increment` button by searching for the `#event/click` record, where the element attribute is the button: 
+
+```eve
+search
+  event = [#event/click element: [#increment]]
+
+commit
+  [#clicked event]
+```
+
+Clicks only last for an instant, but we want to create a permanent record of each click so we can search for them later. This block commits a `#clicked` record that will persist until it's explicitly removed. Much like the `#greeting` text we bound to the `#ui`, variables with the same name are equivalent, so the variable `event` in the `#clicked` record is a reference to the `#event/click` on the `#increment` button.
+
+The identity of a record is determined by its attribute/value pairs. Two records with the same attributes and values are identical in Eve. We included the `event` attribute in the `#clicked` record to differentiate each record. Without this differentiation, we could only ever create a single `#clicked` record. Try removing `event` from the record and click the button to test this out.
+
+## Count the number of clicks
+
+Now let's count the number of times the button has been clicked. Make sure `event` is back in `#clicked`, and then we can count those records directly:
+
+```eve
+search
+  how-many = count[for: [#clicked]]
+
+bind
+  [#ui/text text: "The button has been clicked {{how-many}} times"]
+```
+
+This block searches for every unique `#clicked`, counts them, and returns that value in `how-many`. Then we display this value in a text container using the operator `{{ ... }}`, which inserts the value of the contained variable into the string. An important thing to remember here is that this block will only run when the button has been clicked at least once. Before then, this block will not run because there are no `#clicked` records to count.
+
+## Summary
+
+That's it for the 5 minute introduction to Eve. To summarize:
+
+- Eve programs are made up of blocks.
+- Data are represented by records, key value pairs associated to a unique ID.
+- There are three actions associated with records: search, bind, and commit.
```

Comment:
should we talk about these as sections of a block instead of actions of a record? It seems a little harder to confuse that way.